pkg/virt-controller/watch/snapshot/restore.go

```diff
@@ -733,14 +732,36 @@ func (ctrl *VMRestoreController) getTarget(vmRestore *snapshotv1.VirtualMachineR
 func (ctrl *VMRestoreController) createRestorePVC(
 	vmRestore *snapshotv1.VirtualMachineRestore,
 	target restoreTarget,
-	volumeBackup snapshotv1.VolumeBackup,
-	volumeRestore snapshotv1.VolumeRestore,
+	volumeBackup *snapshotv1.VolumeBackup,
+	volumeRestore *snapshotv1.VolumeRestore,
 	sourceVmName, sourceVmNamespace string,
 ) error {
+	if volumeBackup.VolumeSnapshotName == nil {
```

Comment:
`volumeBackup` is now a pointer, does it need to be null-checked here? Same for `volumeRestore`.

Reply:
Good point, I modified it because I wanted to pass by reference instead of copying the entire object. But that does come with some nil checks.